README.md

```diff
@@ -6,6 +6,78 @@
 
 Django-components is a package that introduces component-based architecture to Django's server-side rendering. It aims to combine Django's templating system with the modularity seen in modern frontend frameworks.
 
+A component in django-components can be as simple as a Django template and Python code to declare the component:
+
+```htmldjango title="calendar.html"
+<div class="calendar">
+  Today's date is <span>{{ date }}</span>
+</div>
+```
+
+```py title="calendar.py"
+from django_components import Component
+
+class Calendar(Component):
+    template_name = "calendar.html"
+```
+
+Or a combination of Django template, Python, CSS, and Javascript:
+
+```htmldjango title="calendar.html"
+<div class="calendar">
+  Today's date is <span>{{ date }}</span>
+</div>
+```
+
+```css title="calendar.css"
+.calendar {
+  width: 200px;
+  background: pink;
+}
+```
+
+```js title="calendar.js"
+document.querySelector(".calendar").onclick = function () {
+  alert("Clicked calendar!");
+};
+```
+
+```py title="calendar.py"
+from django_components import Component
+
+class Calendar(Component):
+    template_name = "calendar.html"
+    js_file = "calendar.js"
```

Comment:
At first I wanted to go with `js_name` and `css_name` as attribute names, so it would be in line with `template` and `template_name`. But TBH, `js_name` doesn't sound intuitive, while `js_file` is more self-explanatory. I'm also considering `js_src`, or open to other ideas.

Reply:
This is tricky. template_name is used for Django's generic views and should be familiar to Django users. I guess the idea is that you just give the file *name*, and then the template finders will expand that to a path, and fetch the actual file. name -> path -> file. In that context, I think js_(file)name makes sense.

But just looking at "js_name", names usually don't map to files in the file system. It could mean a name of a variable, a function, an alias for something. Not knowing that we are referring to a filename likely makes this confusing for beginners. 

And js_filename is too long. So I'm leaning towards js_file, just like you do.

What's left is the inconsistency of template_name and js_file. I think the easiest way forward for that is to add a template_file alias, that does the same thing as template_name. Then things would make sense imho.

Reply:
Good idea, I'll make an alias for `template_file`.